packages/bitcore-node/src/routes/api/tx.ts

```diff
@@ -42,16 +42,35 @@ router.get('/:txId',  async (req, res) => {
   try {
     const tx = await ChainStateProvider.getTransaction({ chain, network, txId });
     if (!tx) {
-      return res.status(404).send(`txid ${txId} could not be found`);
+      return res.status(404).send(`The requested txid ${txId} could not be found.`);
     } else {
-      return res.send([tx]);
+      return res.send(tx);
```

Comment:
@matiu – I'm not sure if you're using this endpoint yet, but just wanted to make sure you saw this. @micahriggan and I couldn't see why this would be wrapped in an array, so I unwrapped it. Is there any reason we'd want it to be an array here?